src/test/java/seatsio/SeatsioClientTest.java

```diff
@@ -52,12 +53,14 @@ protected String createTestChartWithSections() {
         return createTestChart("/sampleChartWithSections.json");
     }
 
-    protected String createTestChartWithErrors() { return createTestChart("/sampleChartWithErrors.json"); }
+    protected String createTestChartWithErrors() {
+        return createTestChart("/sampleChartWithErrors.json");
+    }
 
     protected String createTestChart(String fileName) {
         String testChartJson = testChartJson(fileName);
         String chartKey = randomUUID().toString();
-        stringResponse(post(BASE_URL + "/system/public/" + user.designerKey + "/charts/" + chartKey)
+        stringResponse(Unirest.post(STAGING_BASE_URL + "/system/public/" + user.designerKey + "/charts/" + chartKey)
```

Comment:
Shouldn't this use UnirestUtil.post ?

Reply:
`UnirestUtil.post` sends a basic auth header, but that's not needed here. The `/system/public` URLs contain a public key for authentication.